contrib/gitian/gitian-build.py

```diff
@@ -66,7 +66,7 @@ def rebuild():
         print('\nCompiling ' + args.version + ' ' + os_name)
         infile = 'inputs/monero/contrib/gitian/gitian-' + tag_name + '.yml'
         subprocess.check_call(['bin/gbuild', '-j', args.jobs, '-m', args.memory, '--commit', 'monero='+args.commit, '--url', 'monero='+args.url, infile])
-        subprocess.check_call(['bin/gsign', '-p', args.sign_prog, '--signer', args.signer, '--release', args.version+'-linux', '--destination', '../sigs/', infile])
+        subprocess.check_call(['bin/gsign', '-p', args.sign_prog, '--signer', args.signer, '--release', args.version + os_name, '--destination', '../sigs/', infile])
```

Comment:
This should actually be
````
args.version+'-'+tag_name
````

Reply:
Right, fixed.

Reply:
Still not correct. You left out the '-'.

Reply:
damnit :(